ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -279,3 +275,13 @@ extension CommandBar: UIScrollViewDelegate {
         updateShadow()
     }
 }
+
+/// A UIView subclass that updates its mask frame during layoutSubviews. By default, the layer mask
+/// is not hooked into auto-layout and will not update its frame if its parent frame changes size. This implementation
+/// fixes that.
+private class MaskedView: UIView {
```

Comment:
becauz this is so specific for this file for a particular use case, can we be more clear like CommandingContainerView instead?  "maskedView" makes it little confusing to me at a glance be might think this is a uikit thing?

Reply:
Sure. Updated.